src/actor-widget/actor-widget.html

```diff
@@ -0,0 +1,245 @@
+            <autocomplete service.bind="suggest.straten" value.bind="filters.straat" label="naam">
+              <template replace-part="suggestion">
+                ${suggestion.naam}
+              </template>
+            </autocomplete>
+          </div>
+        </div>
+
+        <div class="large-4 small-9 column">
+          <!-- Huisenumber -->
+          <div class="placeholder-container placeholder">
+            <label for="actor-widget-huisnummer">Huisnummer</label>
+            <autocomplete service.bind="suggest.huisnummer" value.bind="filters.huisnummer" label="naam">
+              <template replace-part="suggestion">
+                ${suggestion.naam}
+              </template>
+            </autocomplete>
+          </div>
+        </div>
+
+        <div class="large-2 small-3 column">
+          <!-- Subadres -->
+          <div class="placeholder-container placeholder">
+            <label for="actor-widget-subadres">Postbus</label>
+            <input type="text" name="actor-widget-subadres" value.bind="filters.subadres" id="actor-widget-subadres">
+          </div>
+        </div>
+
+        <div class="large-6 small-6 column">
+          <!-- Person ID -->
+          <div class="placeholder-container placeholder">
+            <label for="actor-widget-persid">Persoon ID</label>
+            <input type="text" name="actor-widget-persid" value.bind="filters.persid" id="actor-widget-persid">
+          </div>
+        </div>
+
+        <div class="large-6 small-6 column">
+          <!-- RRN -->
+          <div class="placeholder-container placeholder">
+            <label for="actor-widget-rrn">Rijksregisternummer</label>
+            <input type="text" name="actor-widget-rrn" value.bind="filters.rrn" id="actor-widget-rrn">
+          </div>
+        </div>
+
+        <div class="large-6 small-6 column">
+          <!-- KBO -->
+          <div class="placeholder-container placeholder">
+            <label for="actor-widget-kbo">KBO</label>
+            <input type="text" name="actor-widget-kbo" value.bind="filters.kbo" id="actor-widget-kbo">
+          </div>
+        </div>
+
+      </div>
+
+      <!-- RESULT TABLE -->
+      <div class="row">
+        <ag-grid-aurelia 
+          if.bind="showTable"
+          class="large-12 column ag-theme-balham" 
+          grid-size-changed.call="onGridSizeChanged()"
+          column-resized.call="gridOptions.api.resetRowHeights()" 
+          grid-options.bind="gridOptions"
+          grid-ready.call="onGridReady()"
+          row-clicked.call="selectActor($event)">
+      </ag-grid-aurelia>
+      </div>
+      <!-- TABLE ACTIONS -->
+      <div class="row" if.bind="showTable">
+        <div class="large-12 column">
+          <div class="actor-actions">
+            <!-- Create actor -->
+            <div class="actor-action">
+              <a class="actor-toevoegen" target="_blank"
+                href="${scope.actorenApiService.config.actorUrl}/beheer#/actoren/aanmaken">
```

Comment:
@cedrikv in inventaris staat actorUrl, in inventaris actorenUrl. Wat doen we hiermee?